src/api/album.ts

```diff
@@ -0,0 +1,31 @@
+import { get, post } from '@api/base';
+import { WithResponse, AlbumData, AlbumInfo, AlbumCreated } from 'src/types/api';
+import { imageUpload } from './user';
+
+export async function getUserAlbums(userId: number) {
+  return await get(`profile/${userId}/albums`) as WithResponse<{ albums: AlbumData[] }>;
+}
+
+export async function getAlbum(albumID : number) {
```

Comment:
Речь про неоднобразный нейминг
```suggestion
export async function getUserAlbums(userId: number) { // тут userId
  return await get(`profile/${userId}/albums`) as WithResponse<{ albums: AlbumData[] }>;
}

export async function getAlbum(albumId : number) { // а тут было почему то albumID
```

Reply:
Исправил названия